cluster/gce/gci/configure-helper.sh

```diff
@@ -91,7 +93,7 @@ function get-local-disk-num() {
 function safe-block-symlink(){
   device=$1
   interface=$2
-  mkdir -p "/mnt/disks/by-uuid/google-local-ssds-${interface,,}-block"
+  mkdir -p "${UUID_MNT_PREFIX}-${interface,,}-block"
```

Comment:
can you handle lowercasing $interface when we're parsing the args?

Reply:
i mean very technically none of these ever need to be lower cased because they just come from some hardcoded "scsi" "fs" strings. I'll remove them all.